sci-physics/cernlib/cernlib-2023.10.31.0.ebuild

```diff
@@ -0,0 +1,34 @@
+EAPI=8
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake
+
+DESCRIPTION="CERNLIB - CERN's library for data analysis"
+HOMEPAGE="https://cernlib.web.cern.ch/cernlib/"
+SRC_URI="https://cernlib.web.cern.ch/download/2023_source/tar/${P}-free.tar.gz"
+S="${WORKDIR}/${P}-free"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	x11-libs/motif:0
+	virtual/lapack
+	dev-lang/cfortran
+	x11-libs/xbae
+	net-libs/libnsl
+"
+DEPEND="
+	${RDEPEND}
+	x11-misc/imake
+	x11-misc/makedepend
+	virtual/pkgconfig
```

Comment:
`pkgconfig` is a BDEPEND, those other two packages also suspiciously sound like the might be used during building.

Reply:
I should check which of them are still required. They originally are from [cernlib-2006-r9.ebuild](https://gitlab.com/APN-Pucky/gentoo-hep-forge/-/blob/920955d8197ab8e6e1fd34ababd509a7fef9dd35/sci-physics/cernlib/cernlib-2006-r9.ebuild) that I found somewhere on the web, but since they (seemingly) switched from `imake` to `cmake` I should check what is still really required.